pebble.go

```diff
@@ -0,0 +1,481 @@
+//go:build pebbledb
+// +build pebbledb
+
+package db
+
+import (
+	"bytes"
+	"fmt"
+	"path/filepath"
+
+	"github.com/cockroachdb/pebble"
+)
+
+// ForceSync
+/*
+This is set at compile time. Could be 0 or 1, defaults is 0.
+It will force using Sync for NoSync functions (Set, Delete, Write)
+
+Used as a workaround for chain-upgrade issue: At the upgrade-block, the sdk will panic without flushing data to disk or
```

Comment:
How was this handled with existing databases where we did not introduce this `ForceSync` flag?

Reply:
I think that it may make more sense to go through #115 -- it uses pebbledb 1.0 and doesn't seem to need this, but @baabeetaa can provide more context I think.

Reply:
i havent tried newer version of pebbledb. With current version, set it default to 1 instead.

Reply:
btw, i'll test upgrading without ForceSync=1 with new version of pebbledb tomorrow and report.

Reply:
Nice -- could you test the #115 branch?

Reply:
ok, there will be Omniflix mainnet upgrade tomorrow.

Reply:
i tried upgrading Omniflix with pebble v1.0.0. It doesnt work without using ForceSync

Reply:
Imo, using default ForceSync=1 shoulnt be a problem.
https://github.com/cockroachdb/pebble/blob/master/options.go#L356

Reply:
i updated default ForceSync=1, so no need to use this flag when upgrading.